Automattic/woocommerce-payments
# Add support for Croatia

Fixes #5972 

#### Changes proposed in this Pull Request

Add support for Croatia









#### Testing instructions
- On server side checkout 3302-gh-Automattic/woocommerce-payments-server branch
- **Base fees**
- Go to Woo -> Settings -> General and set Store Country to Croatia and currency to Euro
- Remove supported countries from the account on server side
- Place an order - base fees should be applied
- **FX fees**
- change store currency to USD
- Remove supported countries from the account on server side
- Place an order - base and FX fees should be applied
- **EPS fees**
- Place an order with EPS
- You will be redirected to a Stripe test page where you can approve/fail the order
- Approve the order - EPS fees should be applied

Continue with all fees defined for Croatia.





-------------------


**Post merge**
